knowledge_verificator/materials.py

```diff
@@ -89,8 +125,9 @@ def add_material(self, material: Material) -> None:
             ValueError: Raised if title of a learning material is empty.
             FileExistsError: Raised if learning material in a supplied
                 path already exists.
-            ValueError: Raised if a supplied path path is outside the
-                directory for learning materials.
+            ValueError: Raised if a supplied path is outside the
```

Comment:
This case is not implemented :c

Reply:
Also, I would say, the user should have no say in the topic where file should be stored, they only provide material title and other required stuff, and the backend data is being autogenerated.

Reply:
That is right. I will open an issue for this problem.

It is here: #15